README.md

```diff
@@ -27,6 +27,19 @@ Quick note, it is considered a good practice to use Google before contacting men
 
 For any troubles you're having with the organization, mentors or other students, feel free to contact the GCI admins (details below).
 
+## Don’t do these things
+- Don't use foul language to any student or mentor in any case. Always be polite to other community members and mentors.
+- Try to avoid arguing with mentors regarding your task submission.
+- At times there might be delay in reviewing a task .You should *NOT* be constantly texting them to respond.
+- **WARNING**  Never try to copy someone else’s work .
```

Comment:
@RaghavAwasthi Please make it
-**WARNING**  Never try to copy someone else’s work. Plagiarism is strictly prohibited, you will be banned from the program.

Reply:
+1 👍